egi-check-in-validator.spec

```diff
@@ -45,11 +45,14 @@ rm -rf $RPM_BUILD_ROOT
 %files -f INSTALLED_FILES
 %{_sysconfdir}/%{name}/config/egi-check-in-validator.ini
 %{_sysconfdir}/%{name}/config/logger.ini
-%{_localstatedir}/log/%{name}/egi-check-in-validator.log
+%dir %attr(0644, condor, condor) %{_localstatedir}/log/%{name}
+%attr(0644,condor,condor) %{_localstatedir}/log/%{name}/egi-check-in-validator.log
```

Comment:
We can remove the lines about creating the log files if the default is to use syslog